mkt/feed/views.py

```diff
@@ -811,6 +815,35 @@ def get_es_feed_element_query(self, sq, feed_items):
 
         return sq.filter(es_filter.Bool(should=filters))[0:len(feed_items)]
 
+    def _featured_website_query(self, region=None, n=11):
```

Comment:
nitpick, but calling it ```_get_featured_websites``` might make it more clear that it's actually executing the query